pkg/rook/upgrade.go

```diff
@@ -55,6 +55,10 @@ func hasRookOrCephVersion(deployments *appsv1.DeploymentList, desiredVersion str
 	for _, dep := range deployments.Items {
 		rookVer := normalizeRookVersion(dep.Labels[labelKey])
 		if rookVer != desiredVersion {
+			if strings.Contains(rookVer, "0.0.0") || len(rookVer) == 0 {
```

Comment:
i think we should target the exact bug. what is the label? is it empty or v0.0.0-0?